test/test_multiprocessing.py

```diff
@@ -418,9 +433,56 @@ def test_variable_sharing(self):
                            requires_grad=requires_grad)
             self._test_autograd_sharing(var)
 
+    def test_leaf_variable_sharing(self):
+        devices = ['cpu']
+        if torch.cuda.is_available() and not NO_MULTIPROCESSING_SPAWN and TEST_CUDA_IPC:
+            devices.append('cuda')
+        for device in devices:
+            for requires_grad in [True, False]:
+                var = Variable(torch.arange(1., 26, device=device).view(5, 5), requires_grad=requires_grad)
```

Comment:
Avoid using Variable in new code. (Use `torch.arange(1., 26, device=device).view(5, 5).requires_grad_(requires_grad)` instead)